src/tests/test_utils/common_test_utils/src/postgres_helpers.cpp

```diff
@@ -28,12 +28,12 @@ fnPQresultErrorMessage PQresultErrorMessage;
 
 const char* PGPrefix(const char* text, ::testing::internal::GTestColor color) {
 #ifdef __GNUC__
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wformat-security"
+#    pragma GCC diagnostic push
```

Comment:
Why?

Reply:
clang-format changed it automatically. reverted